tests/handlers/koji/test_rebuild_flatpak_application_on_module_ready.py

```diff
@@ -35,6 +38,9 @@ def setUp(self):
         self.advisory = ErrataAdvisory(123, "RHSA-123", "QE", ["module"], "Critical")
         self.from_advisory_id.return_value = self.advisory
         self.handler = RebuildFlatpakApplicationOnModuleReady()
+        self.mock_get_auto_rebuild_image_list = self._patch('freshmaker.handlers.koji.RebuildFlatpakApplicationOnModuleReady._get_auto_rebuild_image_list')
+        self.mock_filter_images_with_higher_rpm_nvr = self._patch('freshmaker.handlers.koji.RebuildFlatpakApplicationOnModuleReady._filter_images_with_higher_rpm_nvr')
```

Comment:
Could you put those 2 lines on multiple lines? For readability.

Reply:
Ok. Let me black it. Maybe there will be some unrelated changes.

Reply:
Ok. Let me black it. Maybe there will be some unrelated changes.